### DISCUSSION TITLE
Auto-generation of R API docs

### DISCUSSION THREAD

**Discussion by johnkerl** [quarto-dev/quarto-cli#6545] [Q&A]
### Description

Apologies in advance if this was discussed at https://github.com/quarto-dev/quarto-cli/discussions/categories/q-a or https://github.com/quarto-dev/quarto-cli/issues (I didn't find priors in my searching so far).

Given an R package and all its `*.Rd` files I want to find a CLI command (ultimately, CI-able) which will create corresponding `.qmd` files that can be listed within a `_quarto.yml`.

[The context is a multi-language repo which will have, in its `_quarto.yml`, sections including Python API (autogenerated from Python docstrings), Python examples (hand-written), R API (autogenerated from R docstrings), and R examples (hand-written vignettes). The question I'm posting here is about the third of those four sections.]

After looking through https://quarto.org/docs/computations/r.html several times I'm seeing that RStudio is the suggested venue -- what I need though is quite simply a command-line ability to produce `.qmd` files.

**Comment by mcanouil** [comment]
See:
- https://github.com/quarto-dev/quarto-cli/discussions/1591
